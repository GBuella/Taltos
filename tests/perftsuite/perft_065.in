setboard 8/8/8/8/8/7K/7P/7k w - - 0 1
echo 8/8/8/8/8/7K/7P/7k w - - 0 1
perft 1
perft 2
perft 3
perft 4
perft 5
perft 6
