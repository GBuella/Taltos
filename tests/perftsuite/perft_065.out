8/8/8/8/8/7K/7P/7k w - - 0 1
3
7
43
199
1347
6249
